sp-docs/docs/automatic_checks_styleguide.rst

```diff
@@ -0,0 +1,58 @@
+.. _automatic-checks-styleguide:
+
+Style guide linting
+===================
+
+The starter pack includes a method to run the `Vale`_ documentation linter configured with `the Vale rules for the current style guide <Vale rules_>`_.
```

Comment:
The current implementation is to fetch and clean up Vale files when used locally. The remote GitHub action workflow currently only uses the remote rules from the style guide repo.

So there's no way to use your own set of rules with our current implementation without adjusting the defaults - in which case you could alter things relatively easily.

We should consider this scenario, and there would be relatively easy ways to implement custom rules, but I believe it would be out of scope for this cycle.

Reply:
That sounds like we don't need an update here, correct?

Reply:
Correct, unless we want to clarify that our current implementation does not provide a framework for custom rules.